CODEOWNERS

```diff
@@ -62,7 +62,9 @@ homeassistant/components/switch/rainmachine.py @bachya
 homeassistant/components/switch/tplink.py @rytilahti
 homeassistant/components/xiaomi_aqara.py @danielhiversen @syssi
 
+homeassistant/components/*/axis.py @kane610
```

Comment:
Is this part of the deconz PR?

Reply:
It got overridden in a separate PR so I just add it again here out of pure laziness